apps/demos/Demos/Menu/Scrolling/description.md

```diff
@@ -1 +1,15 @@
-## Menu scrolling
+The Menu component supports submenu item scrolling. If the combined item length exceeds the screen size, a scrollbar appears, and the submenu height is restricted.
+
+You can use one of the following event handlers to configure submenus:
+
+- [onSubmenuShowing](/Documentation/ApiReference/UI_Components/dxMenu/Configuration/#onSubmenuShowing)
+
+- [onSubmenuShown](/Documentation/ApiReference/UI_Components/dxMenu/Configuration/#onSubmenuShown) 
+
+- [onSubmenuHiding](/Documentation/ApiReference/UI_Components/dxMenu/Configuration/#onSubmenuHiding)
+
+- [onSubmenuHidden](/Documentation/ApiReference/UI_Components/dxMenu/Configuration/#onSubmenuHidden)
+
+These handlers access `submenuContainer`, the root submenu element, and `itemData`, which contains root item and submenu item data.
```

Comment:
If "submenuContainer" and "the root menu element" are the same thing, then I would made the following change.

(Right now, it sounds like `submenuContainer` and "the root submenu elements" are two different items.)

```suggestion
These handlers can access the root submenu element (`submenuContainer`), as well as data from root and submenu items (`itemData`).
```